packages/app-layout/src/styles/vaadin-app-layout-base-styles.js

```diff
@@ -173,7 +173,7 @@ export const appLayoutStyles = css`
   @media (max-width: 800px), (max-height: 600px) {
     :host {
       --vaadin-app-layout-drawer-overlay: true;
-      --_vaadin-app-layout-drawer-width: var(--vaadin-app-layout-drawer-width, 20em);
+      --_vaadin-app-layout-drawer-width: var(--vaadin-app-layout-drawer-width, 320px);
```

Comment:
While I understand the intend for spacing in general, the sizing of the drawer and the min-width of the dialog feel more naturally with em, or? At least in my understanding it would be better that the "canvas" of my "big text" is big enough so that bigger text does not mean instantly that the whole layout has unncessary line breaks

Reply:
I’ll keep it like it is now. Let’s try to remember to test it before v25 final.